src/forms/FormField.js

```diff
@@ -345,69 +360,40 @@ export const styles = {
     left: '0px',
     width: '100%',
     height: 'auto',
-    padding: '10px 20px 20px 0px',
-    backgroundColor: 'white'
-  },
-  saveButton: {
-    display: 'inline-block',
-    fontSize: '11pt',
-    height: '40px',
-    color: 'white',
-    background: '#36B278',
-    border: 'none',
-    borderRadius: '4px',
-    marginTop: '10px',
-    lineHeight: '40px',
-    textAlign: 'center',
-    cursor: 'pointer',
-    padding: '0 20px',
-    marginLeft: '10px',
-    disabled: {
-      background: 'grey'
-    }
-  },
-  cancelButton: {
-    fontSize: '11pt',
-    height: '40px',
-    color: '#777',
-    background: 'white',
-    border: '1px solid #ccc',
-    borderRadius: '4px',
-    marginTop: '10px',
-    lineHeight: '40px',
-    textAlign: 'center',
-    cursor: 'pointer',
-    padding: '0 20px',
-    marginLeft: '10px'
+    padding: '10px 22px 30px 0px',
   },
   label: {
     display: 'block',
     width: '100%',
     marginBottom: '10px'
   },
-  fieldTitle: {
+  inputField: {
+    fontFamily: 'Roboto',
     fontSize: '12pt',
-    padding: '5px 0',
-    width: '75%',
+    width: '50%',
     display: 'block',
     border: 'none',
-    background: 'none'
+    background: 'none',
+    boxSizing: 'border-box',
   },
+  fieldTitle: {
+    width: '75%',
+    fontSize: '12pt',
+    minHeight: '32px',
+    marginTop: '-3px'
+},
   fieldDescription: {
-    fontSize: '11pt',
-    padding: '5px 0',
-    marginBottom: '20px',
-    width: '50%',
-    display: 'block',
-    border: 'none',
-    background: 'none'
+    fontSize: '10pt',
+    width: '100%',
+    minHeight: '40px',
+    marginBottom: '30px'
```

Comment:
pro tip: anytime you reference the string `'30px'`, Radium will also accept the integer `30` and convert it for you. It adds some brevity to our code.